backend/btrixcloud/orgs.py

```diff
@@ -1071,12 +1071,10 @@ async def delete_org_and_data(self, org: Organization, user_manager: UserManager
         await self.crawl_configs_db.delete_many({"oid": org.id})
 
         # Delete profiles
-        async for profile in self.profiles_db.find({"oid": org.id}, projection=["_id"]):
-            await self.profile_ops.delete_profile(profile["_id"], org)
+        await self.profile_ops.delete_org_profiles(org.id)
 
         # Delete collections
-        async for coll in self.colls_db.find({"oid": org.id}, projection=["_id"]):
-            await self.coll_ops.delete_collection(coll["_id"], org)
```

Comment:
Calling `delete_collection` also fired the deleted notification webhook - do we want to do something different for that? not fire any webhooks or a delete all collections? might be nice to keep it as is, though it can increase time that it takes to delete.

Reply:
Ah right, guess there was a reason things were the way they were! I'll revert my last commit